### DISCUSSION TITLE
Welcome to Cryengine-Converter Discussions!

### DISCUSSION THREAD

**Discussion by Markemp** [Markemp/Cryengine-Converter#77] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by coloradocomputerpro** [comment]
Hi, I am James and I am a non recovering Star Citizen'aholic...

**Reply by Markemp** [reply]
Well, that was quick! :)  

I was just going over some best practices for repos and I didn't even know there was a discussions section.  This is great because there are some issues that I feel need more discussions, but having them in the Issues section just adds bloat to the issues and makes it hard to stay focused on the actual problem.  Things like how to calculate normals from the tangents/bitangents, or how to handle materials better.

Thanks for posting, and your help with the project!

**Comment by MadochGreyson** [comment]
Hi!
First, thank you for making the converter...it's amazing.  I've been building out models from the holo-viewer, and this process is much more detailed.
Question: When I run the bulk converter process, after looking through .skins the program seems to just stop and not close down.  Is that normal?
Thanks again,
-Madoch

**Reply by Markemp** [reply]
CIG recently (relatively) introduced a new game data file (referred to as #ivo files, after the developer Ivo at CIG) that deviates significantly from standard Cryengine files.  I'm still trying to sort out some of the data structures, and until this is done it does cause the program to loop endlessly.  Super annoying, but there are a few Issues open on it right now.

Until this is complete, there is a chance that some files will not process. :(

**Comment by MadochGreyson** [comment]
Thank you!  Glad to know it's not just me.  :-)
-Steve

________________________________
